test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -86,6 +92,7 @@
 	private static final String[] TEST_ARG_LIST = {"-Dtest.option1=testJavaToolOptions1", "-Dtest.option2=testJavaToolOptions2", 
 			"-Dtest.option3=testJavaToolOptions3", "-Dtest.option4=testJavaToolOptions4"};
 	private static final String OPTIONS_FILE_SUFFIX = ".test_options_file";
+	private static final String cmdlineArg = "-DcmdlineArg";
```

Comment:
Why the different naming convention for this `static final String`?  All the others are uppercase

Reply:
Fixed.